packages/hub/indexers.js

```diff
@@ -134,10 +134,10 @@ class Indexers extends EventEmitter {
           // Compare each branch, so we don't invalidate the schemas
           // unnecessarily
           for (let [branch, newSchema] of Object.entries(schemas)) {
-            let oldSchema = priorCache[branch];
+            let oldSchema = (await priorCache)[branch];
             if (!newSchema.equalTo(oldSchema)) {
               log.info('schema for branch %s was changed', branch);
-              priorCache[branch] = newSchema;
+              (await priorCache)[branch] = newSchema;
```

Comment:
This kind of bug is something TypeScript would actually catch for us. And the build tooling (especially for monorepos) has gotten a lot better now. Perhaps it's time.

Reply:
I don't disagree :-)